editor/editor_inspector.cpp

```diff
@@ -784,6 +786,30 @@ void EditorProperty::_gui_input(const Ref<InputEvent> &p_event) {
 			update();
 			emit_signal(SNAME("property_checked"), property, checked);
 		}
+	} else if (mb.is_valid() && mb->is_pressed() && mb->get_button_index() == MOUSE_BUTTON_RIGHT) {
+		_ensure_popup();
+		menu->set_position(get_global_transform().xform(get_local_mouse_position()));
```

Comment:
This isn't correct way to set popup position anymore. Use `menu->set_position(get_screen_position() + get_local_mouse_position());` instead.